src/WebJobs.Script/Workers/ProcessManagement/JobObjectRegistry.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.Azure.WebJobs.Script.Workers
 {
     // Registers processes on windows with a job object to ensure disposal after parent exit
-    internal class JobObjectRegistry : IProcessRegistry, IDisposable
```

Comment:
This is removed because JobObjectRegistry is disposing the worker process early (due to race condition). Code is required to wait for few seconds for graceful termination of worker before killing the process.

Reply:
Do we still get to dispose the JobObjectRegistry at some point or we no longer need to?